test/tests/mesh/add_sideset_ids/tests

```diff
@@ -24,4 +24,11 @@
     design = MooseMesh.md
     requirement = 'The system shall allow specifying valid sideset names that do not exist on the initial mesh, and assign IDs automatically.'
   []
+  [remove_sidesets]
+    type = CSVDiff
+    input = remove_sideset_ids.i
+    csvdiff = remove_sideset_ids_out.csv
+    design = MooseMesh.md
+    requirement = 'The system shall remove specifying sidesets that exist on the initial multiple blocks when elements are moved to new blocks.'
```

Comment:
```suggestion
    requirement = 'The system shall be able to remove sides from sidesets when these sides' elements are moved to new blocks.'
```
or do you mean straight up deleting the sideset?

Reply:
The sidesets are only associated with the subdomain the user chooses.  The elements that are moved to a new subdomain will no longer have that sideset associated with it.